test/support/assert_async.ex

```diff
@@ -0,0 +1,42 @@
+defmodule AssertAsync do
```

Comment:
There is also a variant of this in the `assertions` package on hex, if you didn't try it yet and hate it